syntax = "proto3";

package helloworld;

service Greeter {
    rpc SendMessage (CustomMessage) returns (HelloReply) {}
}

message CustomMessage {
    int32 user_id = 1;
    string content = 2;
}

message HelloReply {
    string message = 1;
}
